src/main/java/com/coreos/jetcd/KV.java

```diff
@@ -20,36 +20,72 @@
 @Beta
 public interface KV {
 
-  // ***************
-  // Op.PUT
-  // ***************
-
+  /**
+   * put a key-value pair into etcd.
+   *
+   * @param key key in ByteSequence
+   * @param value value in ByteSequence
+   * @return PutResponse
+   */
   CompletableFuture<PutResponse> put(ByteSequence key, ByteSequence value);
 
+  /**
+   * put a key-value pair into etcd.
+   *
+   * @param key key in ByteSequence
+   * @param value value in ByteSequence
+   * @param option PutOption
+   * @return PutResponse
+   */
   CompletableFuture<PutResponse> put(ByteSequence key, ByteSequence value, PutOption option);
 
-  // ***************
-  // Op.GET
-  // ***************
-
+  /**
+   * retrieve keys. By default, get will return the value for "key".
```

Comment:
I copied it from go client. I think it means that without option, the get just return value that's corresponding to the key.

```go
	// Get retrieves keys.
	// By default, Get will return the value for "key", if any.
	// When passed WithRange(end), Get will return the keys in the range [key, end).
	// When passed WithFromKey(), Get returns keys greater than or equal to key.
	// When passed WithRev(rev) with rev > 0, Get retrieves keys at the given revision;
	// if the required revision is compacted, the request will fail with ErrCompacted .
	// When passed WithLimit(limit), the number of returned keys is bounded by limit.
	// When passed WithSort(), the keys will be sorted.
	Get(ctx context.Context, key string, opts ...OpOption) (*GetResponse, error)
```

https://github.com/coreos/etcd/blob/master/clientv3/kv.go#L38-L39